canonical-web-and-design/snapcraft.io
# Move scripts loading from bottom of the page to `<head>`

Follow up to #2292 (merge that one first)

Fixes #2287 
Moves loading scripts (public.js publisher.js, etc) from the bottom of the body to head.

### QA
- ./run or demo
- make sure scripts are loaded early
- make sure pages still work, there are no errors in console (check EVERYTHING!)
